cms/templates/js/xblock-access-editor.underscore

```diff
@@ -0,0 +1,5 @@
+<div class="incontext-editor-action-wrapper">
+    <a href="" class="unit-container visibility-button" title="<%- gettext('Access Settings') %>">
```

Comment:
It is my understanding that neither is OK. See #12993 for where usages of the a-tag were converted to buttons in the LMS.

I would double-check with @cptvitamin, but he appears to be on vacation.